src/vma/sock/sockinfo.cpp

```diff
@@ -36,6 +36,7 @@
 #include <sys/epoll.h>
 #include <netdb.h>
 #include <linux/sockios.h>
+#include <sys/eventfd.h>
```

Comment:
may be not needed to include `<sys/eventfd.h>` and `<sys/epoll.h>`

Reply:
Yes, thanks, deleted it